### DISCUSSION TITLE
[error] gdbstub for Arm64

### DISCUSSION THREAD

**Discussion by rootkiter** [utmapp/UTM#5110] [UTM Help (macOS)] [answered]
HOST: Mac mini  2023 16G+256G
Chip. : Apple M2
macOS: Ventura 13.2.1
UTM Version: 4.1.6
VM: Debian-Arm64 Linux 5.10.162

The screenshot is as follows.

![image](https://user-images.githubusercontent.com/6734184/224278510-82ac61ab-d5ba-4b7d-a9ac-38a1fa8160e0.png)

**Comment by osy** [comment, marked as answer]
You have to disable hypervisor in QEMU settings
